shardingsphere-test/shardingsphere-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/asserts/segment/expression/ExpressionAssert.java

```diff
@@ -378,8 +379,12 @@ public static void assertExpression(final SQLCaseAssertContext assertContext,
             ProjectionAssert.assertProjection(assertContext,
                     (ExpressionProjectionSegment) actual, expected.getExpressionProjection());
         } else if (actual instanceof AggregationProjectionSegment) {
+            ExpectedAggregationProjection expectedAggregationProjection = expected.getAggregationProjection();
```

Comment:
Please move this logic to assertProjection.

Reply:
This logic is for get expected unit test object, should not move to assertProjection.